project-archive/multiary/test-files/cmd/slow/ServiceMultiaryPoiIngestCmdTest.sh.stdout.failed

```diff
@@ -0,0 +1,162 @@
+
+MULTIARY INGEST - DELETING REFERENCE LAYER...
```

Comment:
`.stdout.failed` files shouldn't be in the repo.  They should be either copied to the `.stdout` file or the test should be fixed.